samples/name_domains_values/Coded_Domain_Reliability.csv

```diff
@@ -1,7 +1,7 @@
 Name,Value
-Completely Reliable,A
-Usually Reliable,B
-Fairly Reliable,C
-Not Usually Reliable,D
-Unreliable,E
-Reliability Cannot Be Judged,F
+"Completely Reliable",A
```

Comment:
Not sure what caused the quotes to get added in all these new domain versions - intentional?  I don't think this will effect the domain import tools (which can likely handle this excel-like format) - but if this is a global change it may break some of my stuff (I don't think I have tested this format)